src/commands/derive_parser.rs

```diff
@@ -0,0 +1,27 @@
+use crate::utils::auth_token::AuthToken;
+use crate::utils::value_parsers::kv_parser;
+use clap::{command, value_parser, ArgAction::SetTrue, Parser, Subcommand};
+
+#[derive(Parser)]
+pub(super) struct SentryCLI {
+    #[command(subcommand)]
+    pub(super) command: SentryCLICommand,
+
+    #[arg(global=true, long="header", value_name="KEY:VALUE", value_parser=kv_parser, help="Custom headers that should be attached to all requests{n}in key:value format")]
+    pub(super) headers: Vec<(String, String)>,
```

Comment:
I wonder whether we can use `Vec<&'static str, &'static str>` here, since I would expect the CLI arguments to outlive the program. But maybe not, idk